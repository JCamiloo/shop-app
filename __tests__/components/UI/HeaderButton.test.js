import React from "react";
import HeaderButton from "../../../components/UI/HeaderButton";
import renderer from "react-test-renderer";
import { HeaderButtons, Item } from "react-navigation-header-buttons";

describe("HeaderButton component tests", () => {
  test("HeaderButton component renders correctly", () => {
    const tree = renderer
      .create(
        <HeaderButtons HeaderButtonComponent={HeaderButton}>
          <Item title="Add Place" />
        </HeaderButtons>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
