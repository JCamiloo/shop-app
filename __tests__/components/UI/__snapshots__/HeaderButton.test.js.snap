// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderButton component tests HeaderButton component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      Object {
        "marginRight": 5,
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={false}
    hitSlop={
      Object {
        "bottom": 5,
        "left": 5,
        "right": 5,
        "top": 5,
      }
    }
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "opacity": 1,
      }
    }
  >
    <View>
      <Text
        style={
          Array [
            Object {
              "fontSize": 17,
              "marginHorizontal": 10,
            },
            Object {
              "color": "#C2185B",
            },
            undefined,
          ]
        }
      >
        Add Place
      </Text>
    </View>
  </View>
</View>
`;
